app/src/main/java/com/google/samples/apps/sunflower/GardenFragment.kt

```diff
@@ -23,13 +23,26 @@ import android.view.ViewGroup
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.selection.SelectionPredicates
+import androidx.recyclerview.selection.SelectionTracker
+import androidx.recyclerview.selection.StableIdKeyProvider
+import androidx.recyclerview.selection.StorageStrategy
 import com.google.samples.apps.sunflower.adapters.GardenPlantingAdapter
+import com.google.samples.apps.sunflower.adapters.GardenPlantingDetailsLookup
 import com.google.samples.apps.sunflower.databinding.FragmentGardenBinding
 import com.google.samples.apps.sunflower.utilities.InjectorUtils
 import com.google.samples.apps.sunflower.viewmodels.GardenPlantingListViewModel
 
 class GardenFragment : Fragment() {
 
+    private lateinit var selectionTracker: SelectionTracker<Long>
+    private val selectionObserver = object : SelectionTracker.SelectionObserver<Long>() {
```

Comment:
If I do, I will extract this to a class, initialize it just right now after the `selectionTracker` is available.

Reply:
Done.